src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarHeader.Read.cs

```diff
@@ -69,145 +69,98 @@ internal bool TryGetNextHeader(Stream archiveStream, bool copyData)
             }
         }
 
-        // Reads the elements from the passed dictionary, which comes from the first global extended attributes entry,
-        // and inserts or replaces those elements into the current header's dictionary.
-        // If any of the dictionary entries use the name of a standard attribute (not all of them), that attribute's value gets replaced with the one from the dictionary.
-        // Unlike the historic header, numeric values in extended attributes are stored using decimal, not octal.
-        // Throws if any conversion from string to the expected data type fails.
-        internal void ReplaceNormalAttributesWithGlobalExtended(IReadOnlyDictionary<string, string> gea)
-        {
-            // First step: Insert or replace all the elements in the passed dictionary into the current header's dictionary.
-            foreach ((string key, string value) in gea)
-            {
-                _extendedAttributes ??= new Dictionary<string, string>();
-                _extendedAttributes[key] = value;
-            }
-
-            // Second, find only the attributes that make sense to substitute, and replace them.
-            if (gea.TryGetValue(PaxEaATime, out string? paxEaATime))
-            {
-                if (TarHelpers.TryConvertToDateTimeOffset(paxEaATime, out DateTimeOffset aTime))
-                {
-                    _aTime = aTime;
-                }
-            }
-            if (gea.TryGetValue(PaxEaCTime, out string? paxEaCTime))
-            {
-                if (TarHelpers.TryConvertToDateTimeOffset(paxEaCTime, out DateTimeOffset cTime))
-                {
-                    _cTime = cTime;
-                }
-            }
-            if (gea.TryGetValue(PaxEaMTime, out string? paxEaMTime))
-            {
-                if (TarHelpers.TryConvertToDateTimeOffset(paxEaMTime, out DateTimeOffset mTime))
-                {
-                    _mTime = mTime;
-                }
-            }
-            if (gea.TryGetValue(PaxEaMode, out string? paxEaMode))
-            {
-                _mode = Convert.ToInt32(paxEaMode);
-            }
-            if (gea.TryGetValue(PaxEaUid, out string? paxEaUid))
-            {
-                _uid = Convert.ToInt32(paxEaUid);
-            }
-            if (gea.TryGetValue(PaxEaGid, out string? paxEaGid))
-            {
-                _gid = Convert.ToInt32(paxEaGid);
-            }
-            if (gea.TryGetValue(PaxEaUName, out string? paxEaUName))
-            {
-                _uName = paxEaUName;
-            }
-            if (gea.TryGetValue(PaxEaGName, out string? paxEaGName))
-            {
-                _gName = paxEaGName;
-            }
-        }
-
```

Comment:
I removed this code after manually testing the CLI gnu `tar` tool confirming discovering that GEA attributes do not get propagated from the GEA entry into the EA entries that precede the normal entries. They are kept completely separated.

This aligns with the new GEA tar entry we recently approved: The user will be able to read such entries (there can be more than one), and will be free decide what to do with the attributes and how to affect subsequent entries with it.